CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to creature-pigeon
+
+Welcome to **creature-pigeon**! We appreciate your interest in contributing. By participating in this project, you agree to abide by the [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+## How to Contribute
+
+Contributions to **creature-pigeon** can come in many forms, including but not limited to:
+
+- Reporting bugs
+- Suggesting enhancements
+- Writing code patches
+- Improving documentation
+- Providing translations
+
+To contribute, follow these steps:
+
+1. **Fork** the repository and **clone** your fork locally.
+2. **Create a new branch** from `main` for your changes.
+3. **Install dependencies** if necessary (`npm install`).
+4. **Make your changes** and **test** thoroughly.
+5. **Commit** your changes with a descriptive commit message.
+6. **Push** your branch to your fork on GitHub.
+7. **Submit a pull request** (PR) to the `main` branch of the original repository.
+
+## Code Style
+
+- Follow the existing code style and conventions used in the project.
+- Ensure your code passes linting (`npm run lint`) and tests (`npm test`).
```

Comment:
No such thing in repo.

I raised an interesting issue for making shoveler fix that